tests/EdFi.Admin.DataAccess.IntegrationTests/Models/UserContextMappingTests.cs

```diff
@@ -9,8 +9,7 @@
 using NUnit.Framework;
 using Shouldly;
 using System;
-using System.Data.Entity;
-using System.Data.Entity.Migrations;
+using Microsoft.EntityFrameworkCore;
```

Comment:
Reorder using statements to get System.* on top and the rest of the libraries in alphabetic order.